doc/manual/manual.tex

```diff
@@ -6656,7 +6656,71 @@ \subsubsection{Crustal deformation}
 \label{fig:ext3D}
 \end{figure}
 
+\newpage
 
+\subsubsection{Continental extension}
+\label{sec:cookbooks-continental-extension}
+\textit{This section was contributed by John Naliboff}
+
+In the crustal deformation examples above, the viscosity depends solely on the drucker prager yield criterion defined by the cohesion and internal friction angle.  While this approximation works reasonably well for the uppermost crust, deeper portions of the lithosphere may undergoe either brittle or viscous deformation, with the latter depending on a combination of composition, temperature, pressure and strain-rate.  In effect, a combination of the Drucker Prager and Diffusion dislocaton material models is required.  The Visco Plastic material model is designed to take into account both brittle (plastic) and non-linear viscous deformation, thus providing a template for modeling complex lithospheric processes. 
+
+\lstinputlisting[language=prmfile]{cookbooks/continental_extension/continental_extension_material_model.prm}
+
+The cookbook below provides one such example where the continental lithosphere undergoes extension.  Notably, the model design follows that of numerous previously published continental extension studies  \cite[and references therein]{Hui11,Bru14,Nal15}.
+
+\paragraph{Continental Extension}
+The 2D cartesian model spans 400 (x) by 100 (y) km and has a finite element grid with uniform 2 km spacing. Unlike the crustal deformation cookbook, the mesh is not refined with time.
+
+\lstinputlisting[language=prmfile]{cookbooks/continental_extension/continental_extension_geometry_mesh.prm}
+
+
+Similar to the crustal deformation examples above, this model contains a free surface.  Deformation is driven by constant horizontal (x-component) velocities (0.25 cm/yr) on the side boundaries (y-velocity component unconstrained), while the bottom boundary has vertical inflow to balance the lateral outflow.  The top and bottom boundaries have fixed temperatures, while the sides are insulating (zero heat flux).
```

Comment:
Is this actually true with the heat flux? I mean, you are transporting material through these boundaries, so presumably some thermal energy also gets transported through them.